api/package.json

```diff
@@ -42,20 +42,16 @@
     "@ai-sdk/anthropic": "^0.0.51",
     "@ai-sdk/mistral": "^0.0.42",
     "@ai-sdk/openai": "^0.0.66",
+    "@anthropic-ai/sdk": "^0.24.3",
     "@clack/core": "^0.3.4",
     "@clack/prompts": "^0.7.0",
-    "@neondatabase/api-client": "^1.10.3",
-    "giget": "^1.2.3",
```

Comment:
do not remove any of the deps that the `cli` folder has - we need all of the cli deps copied here for  the create flow to work. it's hack, i know, and we should just move the cli folder into `api` if we wanna keep `create` here

Reply:
oops wait i see they were just reordered. closing